plugins.json

```diff
@@ -149,6 +149,18 @@
 			"author": "ZeroPoke",
 			"minCrankshaftVersion": "0.2.2",
 			"source": "https://github.com/ZeroPoke/crankshaft-roundeck"
+		},
+		{
+			"id": "vibrant-crankshaft",
```

Comment:
The folder in your archive (currently `vibrantCrankshaft`) should match your plugin ID (`vibrant-crankshaft`).

Sorry, I haven't documented this stuff yet :(

(If you have Go installed you can try `make run` and it'll run the plugin repo's build process, which will catch some stuff like this and checksum mismatches).

Reply:
Updated. Thanks!